regression/regression/result.txt

```diff
@@ -9099,12 +9272,12 @@ ERROR: IsEnd() is invalid as Begin() is never called
     GetLocation():       33417
     GetEngineType():     153
     GetUnitNumber():     1
-    GetAge():            0
+    GetAge():            1
```

Comment:
Why are these (and the following) apparently unrelated changes to do with age necessary?

Reply:
A guess could be that the number of operations before the AI is suspended and another game tick simulated is reached with the new tests inserted.